atmo/templates/atmo/dashboard.html

```diff
@@ -60,35 +60,24 @@
           <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> Schedule
         </a>
       </h2>
-
     </div>
-    <table class="table table-striped">
-      <thead>
-        <tr>
-          <th>Identifier</th>
-          <th>Result visibility</th>
-          <th>Cluster size</th>
-          <th>Run interval</th>
-          <th>Job timeout</th>
-          <th>Start date</th>
-          <th>End date</th>
-        </tr>
-      </thead>
-      <tbody>
-        {% for spark_job in spark_jobs %}
-        <tr>
-          <td><a href="{{ spark_job.get_absolute_url }}">{{ spark_job.identifier }}</a></td>
-          <td>{{ spark_job.result_visibility }}</td>
-          <td>{{ spark_job.size }}</td>
-          <td>{{ spark_job.get_interval_in_hours_display }}</td>
-          <td>{% if spark_job.job_timeout %}{{ spark_job.job_timeout }}h{% else %}n/a{% endif %}</td>
-          <td>{{ spark_job.start_date }}</td>
-          <td>{{ spark_job.end_date|default:"n/a" }}</td>
-        </tr>
-        {% endfor %}
-      </tbody>
-    </table>
-    {% if not spark_jobs %}<p class="text-left">No scheduled jobs to show.</p>{% endif %}
+    {# We only show the tab navigation if the user is a sparkjob maintainer #}
+    {% if is_sparkjob_maintainer %}
+      <ul class="nav nav-tabs" role="tablist">
+        <li role="presentation" class="active"><a href="#my-jobs" aria-controls="my-jobs" role="tab" data-toggle="tab">My jobs</a></li>
+        <li role="presentation"><a href="#all-jobs" aria-controls="all-jobs" role="tab" data-toggle="tab">All jobs</a></li>
+      </ul>
+    {% endif %}
+    <div class="tab-content">
+      <div class="tab-pane active" id="my-jobs" role="tabpanel">
+        {% include "atmo/_jobs_table.html" with spark_jobs=user_spark_jobs %}
+      </div>
+      {% if is_sparkjob_maintainer %}
+        <div class="tab-pane" id="all-jobs" role="tabpanel">
+          {% include "atmo/_jobs_table.html" with spark_jobs=group_spark_jobs %}
```

Comment:
To clarify, the view would continue to just pass in `spark_jobs` but depending on a GET parameter `jobs` the value would be different. E.g. `/?jobs=all` would mean the `spark_jobs` variable would make the query for all user jobs.